package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "lint": "eslint . --ext .ts",
-    "start": "tsc && node dist/index.js"
+    "start": "tsc && node dist/index.js",
+    "compile": "./node_modules/typescript/bin/tsc"
```

Comment:
I don't have typescript installed globally on my machine so this command was handy to compile after code changes